src/checkout-buttons/strategies/googlepay/googlepay-button-strategy.ts

```diff
@@ -100,6 +100,10 @@ export default class GooglePayButtonStrategy implements CheckoutButtonStrategy {
             return options.googlepaystripe;
         }
 
+        if (options.methodId === 'googlepaystripeupe' && options.googlepaystripeupe) {
```

Comment:
Why not use the method defined over here https://github.com/bigcommerce/checkout-sdk-js/pull/1333/files#diff-1c030e40842d05c5b7e7d0f4727ca2e3394c61401da70a728c0db2665bb597b3R12?